compiler/aarch64/codegen/OMRTreeEvaluator.cpp

```diff
@@ -5520,15 +5520,44 @@ TR::Register *commonLoadEvaluator(TR::Node *node, TR::InstOpCode::Mnemonic op, i
 {
     TR::Symbol *sym = node->getSymbolReference()->getSymbol();
     bool needSync = cg->comp()->target().isSMP() && sym->isAtLeastOrStrongerThanAcquireRelease();
+    bool canUseLDAR = tempMR->getUnresolvedSnippet() == NULL && size <= 8;
```

Comment:
Build failed due to an error at this line:

```
/Users/omr/workspace/Build/compiler/aarch64/codegen/OMRTreeEvaluator.cpp:5523:23: error: use of undeclared identifier 'tempMR'
    bool canUseLDAR = tempMR->getUnresolvedSnippet() == NULL && size <= 8;
```

Reply:
🤦‍♂️ fixed